### PR TITLE
A bunch of fixes/improvements to the services startup (docker compose file) + lil more fixes to the environment + minimal index page

### PR DESCRIPTION
**Pull request by eskozloi**

### CODE REVIEW

File: packages/astro-ui/.prettierignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
 dist
 node_modules
 public
-android
-ios
```
Comment (e3stpavel) [inline packages/astro-ui/.prettierignore]: lmao

File: packages/astro-ui/astro.config.ts [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
-  ],
-
+  // https://unocss.dev/integrations/astro
+  unocss({
+    injectReset: '@unocss/reset/tailwind-compat.css'
+  }), vue()],
   // https://docs.astro.build/en/reference/configuration-reference/#trailingslash
   trailingSlash: 'never'
-})
+});
```
Comment (e3stpavel) [inline packages/astro-ui/astro.config.ts:22]: Why this file has different styling?

**Review by e3stpavel — APPROVED** [review]
Great! I will adress this minor issues and merge it into `dev`

File: packages/astro-ui/astro.config.ts (hunk 2 above)
Comment (eskozloi) [reply packages/astro-ui/astro.config.ts:22]: just forgot to format all the files

**Pull request merged**